docs/CONTRIBUTING.md

```diff
@@ -5,10 +5,9 @@ Want to contribute to the Vignette wiki? Great!
 Before that, keep in mind a couple of points.
 
 1. We use [MKDocs](https://www.mkdocs.org) as our wiki framework.
-
```

Comment:
Keep the blank lines between bullet points as they help editing. This isn't minified CSS where everything is crammed.